### PR TITLE
README updates

### PR DESCRIPTION
**Pull request by amrathesh**
- updated SIE ACS verification guide.
- updated README.md wth SIE ACS instructions.
- updated init_call debug instruction in IR README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,9 +26,10 @@ Navigate to the ES, IR, or SR band for further details on specific scripts and p
 * [LS](./LS)
 
 ## SystemReady Security Interface Extension:
-The SystemReady Security Interface Extension tests if the firmware meets the requirements specified by the Arm [Base Boot Security Requirements specification](https://developer.arm.com/documentation/den0107/latest) (BBSR). Security Interface Extension ACS is integrated in SystemReady SR, ES and IR ACS prebuilt images.
+The SystemReady Security Interface Extension certifies that firmware meets the requirements specified by the Arm [Base Boot Security Requirements specification](https://developer.arm.com/documentation/den0107/latest) (BBSR). The Security Interface Extension ACS is now integrated to SystemReady SR, ES and IR bands, but not the LS band.
```
Comment (edhay) [inline README.md:29]: Revert the change to old.

File: common/docs/SIE_ACS_Verification.md [hunk 2]
```diff
@@ -45,7 +45,17 @@ cd ..
 rm -rf swtpm/ libtpms/
 ```
 
-## Building UEFI Firmware
+## Running SIE ACS on QEMU with UEFI firmware
+
+### Build QEMU model
+Follow build instructions from https://www.qemu.org/download/#source
+
+Note: During configure stage, enable slirp library build by appending ./configure with --enable-slirp
```
Comment (edhay) [inline common/docs/SIE_ACS_Verification.md:53]: it is a networking support library. Since run command as netdev it is required.

File: common/docs/SIE_ACS_Verification.md [hunk 3]
```diff
@@ -85,7 +97,7 @@ truncate -s 64M flash1.img
 QEMU=<path to QEMU model>
 FLASH0=<path to flash0.img>
 FLASH1=<path to flash1.img>
-IMG=<path to systemready IR/ES/SR image>
+IMG=<path to systemready ES/S/IR image>
```
Comment (edhay) [inline common/docs/SIE_ACS_Verification.md:100]: SR
Remove IR, as it is not a general use case

IR image also be run if the firmware is UEFI using the above steps.

File: README.md (hunk 1 above)
Comment (amrathesh) [reply README.md]: Done

File: common/docs/SIE_ACS_Verification.md (hunk 2 above)
Comment (amrathesh) [reply common/docs/SIE_ACS_Verification.md]: Added a note.

File: common/docs/SIE_ACS_Verification.md (hunk 3 above)
Comment (amrathesh) [reply common/docs/SIE_ACS_Verification.md]: Added a note.

**Review by edhay — APPROVED** [review]
